README.md

```diff
@@ -128,7 +128,7 @@ The following is a complete example from our [`templates/dendritic`](https://git
 
 - Includes flake-parts-builder's `_bootstrap.nix`.
 - Uses bootstrap to load parts from ./flake-parts
-- Uses bootstrap to load ./flake-parts/\_meta as flake-file configs.
+- Uses bootstrap to load ./flake-parts/_meta as flake-file configs.
```

Comment:
the [md formatter](https://github.com/vic/flake-file/actions/runs/19150941951/job/54740776558?pr=41#step:6:241) needs a backslash here.